Makefile

```
@@ -83,14 +85,19 @@ test-data-conda-dot-conda-packages:
 		'./tests/data/*-0.tar.bz2' \
 		'.conda'
 
+$(NUMPY_WIN_DEBUG_WHL):
+	curl \
+		https://files.pythonhosted.org/packages/be/b0/611101990ddac767e54e2d27d1f4576ae1662cca64e2d55ef0e62558ec26/numpy-1.26.3-cp310-cp310-win_amd64.whl \
+		-o tests/data/numpy-1.26.3-cp310-cp310-win_amd64.whl
```

```suggestion
		-o $(NUMPY_WIN_DEBUG_WHL)
```

Let's re-use this variable, to be sure these 2 paths stay in sync.